### DISCUSSION TITLE
Why is there no button to delete a project?

### DISCUSSION THREAD

**Discussion by mrgrzanka** [community/community#12927] [Projects and Issues] [answered]
Is there any option to delete project?

**Comment by 2percentsilk** [comment, marked as answer]
We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need. Give it a try and let us know how it goes 😄
